src/components/IndividualArticle.jsx

```diff
@@ -0,0 +1,34 @@
+import { useParams } from "react-router-dom";
+import { fetchIndividualArticle } from "../utils/apicalls";
+import { useEffect, useState } from "react";
+
+
+export const IndividualArticle = () => {
+const {article_id} = useParams()
+const [article, setArticle] = useState([])
+const [loading, setLoading] = useState(false)
```

Comment:
There is a little too much switching between true/false for your loading state. 

You're choosing to have the initial value of loading as `false` - but then immediately switch to true in your useEffect, this could be simplified by always assuming you'll be loading when this component initially renders.